guides/release/components/built-in-components.md

```diff
@@ -127,18 +191,36 @@ The modern, Octane-style way to handle keyboard events is to [write a modifier](
 
 There are [community-made addons](https://emberobserver.com/?query=keyboard) to help manage keyboard events. For example, with [ember-keyboard](https://github.com/adopted-ember-addons/ember-keyboard), you can write,
 
-```handlebars
-{{!-- Before --}}
-<Input
-  @enter={{this.doSomething}}
-  @escape-press={{this.doSomethingElse}}
-/>
-
-{{!-- After --}}
-<Input
-  {{on-key "Enter" this.doSomething}}
-  {{on-key "Escape" this.doSomethingElse event="keydown"}}
-/>
+```gjs
+import Component from "@glimmer/component";
+import { Input } from '@ember/component';
+import onKey from 'ember-keyboard/modifiers/on-key.js';
```

Comment:
question: Do we want to include file extensions in imports?

Would be something to align across all docs.